### PR TITLE
Move to production

### PR DESCRIPTION
**Pull request by NoahNsimbe**
- [x] https://github.com/airqo-platform/AirQo-api/pull/1932
- [x] https://github.com/airqo-platform/AirQo-api/pull/1935
- [x] https://github.com/airqo-platform/AirQo-api/pull/1936
- [x] https://github.com/airqo-platform/AirQo-api/pull/1934
- [x] https://github.com/airqo-platform/AirQo-api/pull/1922
- [x] https://github.com/airqo-platform/AirQo-api/pull/1938
- [x] https://github.com/airqo-platform/AirQo-api/pull/1941
- [x] https://github.com/airqo-platform/AirQo-api/pull/1940
- [x] https://github.com/airqo-platform/AirQo-api/pull/1942
- [x] https://github.com/airqo-platform/AirQo-api/pull/1848
- [x] https://github.com/airqo-platform/AirQo-api/pull/1945

### CODE REVIEW

**Comment by Baalmart** [comment]
Can't login via STAGING **but the login endpoint seems to be working fine**. We shall hold this PR as draft until we figure out what is going on. Thanks! @Codebmk / @NoahNsimbe / @BenjaminSsempala 

Screenshot 2023-07-30 at 12 07 30
Screenshot 2023-07-30 at 12 17 57

**Review by Codebmk — APPROVED** [review]
Collocation lgtm @NoahNsimbe

**Comment by Baalmart** [comment]
@BenjaminSsempala , using `staging`, could you look into all endpoints relating to the mobile app? I suspect that something could be off after our recent heavy refactoring.
Screenshot 2023-07-31 at 16 26 02

File: src/auth-service/models/User.js [hunk 1]
```diff
@@ -516,16 +523,12 @@ UserSchema.methods = {
     return jwt.sign(
       {
         _id: this._id,
```
Comment (Baalmart) [inline src/auth-service/models/User.js:525]: @Codebmk , this could be the cause of the issue. Thanks! cc: @BenjaminSsempala

Comment (Baalmart) [reply src/auth-service/models/User.js:525]: A PR is going to be raised asap!

**Pull request merged**